input/socket.js

```diff
@@ -20,8 +20,7 @@ function createTcpServer ({pipeline, name, port, handler}) {
       if (err) {
         pipeline.status(err, 'Cannot start: ' + err.message)
       } else {
-        pipeline.status(null, `Listening on port ${port}.`)
-        console.log(`${name} listening on TCP port ${port}.`)
+        pipeline.status(null, `${name} listening on TCP port ${port}.`)
```

Comment:
Hmmm. I would say we should keep the `Listening on port ${port}` as the status and it is the responsibility of the `pipeline.status` function to add the name in front nope?
As otherwise on the frontend, where we are listing the status, we are first putting the name in a column, then the message, so the name would be two times there.

Reply:
You're absolutely right. Will fix this.